src/basilisp/lang/runtime.py

```diff
@@ -686,6 +734,7 @@ def require(self, ns_name: str, *aliases: sym.Symbol) -> BasilispModule:
             ns_sym = sym.symbol(ns_name)
             ns = self.get(ns_sym)
             assert ns is not None, "Namespace must exist after being required"
+            self.add_alias(ns, ns_sym)
```

Comment:
I think it was always my intent to add the fully-qualified name as an alias to the namespace, but it never mattered before now.